src/labels/CustomLabels.labels

```diff
@@ -16,6 +16,14 @@
         <shortDescription>AfflProgEnrollDeleted</shortDescription>
         <value>If selected, when deleting a Program Enrollment, delete its related Affiliation. Otherwise, keep the Affiliation and set its Status to the value defined in Status Specified for Affiliations Not Deleted.</value>
     </labels>
+    <labels>
```

Comment:
Minor, but I think there should be changes to other translation files as a result of this new label - if we retrieve the metadata from the scratch org, they should get pulled down automatically.